twine/exceptions.py

```diff
@@ -73,3 +73,15 @@ class InvalidSigningConfiguration(TwineException):
     """Both the sign and identity parameters must be present."""
 
     pass
+
+
+class InvalidConfiguration(TwineException):
+    """Raised when configuration is invalid."""
+
+    pass
+
+
+class InvalidDistribution(TwineException):
```

Comment:
This should probably also subclass ``ValueError``.

Reply:
@dstufft Do you think the `InvalidConfiguration` should similarly subclass a `TypeError`?

Reply:
If that's what it used to raise, yea.